### ISSUE TITLE
About the synthetic dataset 

### ISSUE THREAD

**Issue by Wickedness230** [weichen582/RetinexNet#8]
I noticed that there are some deformations in the synthetic image pairs (384×384) provided by your project page. Maybe you did some operations like resize? Do they have any impacts on your results? 
By the way, in your paper, you said that these raw images are resized to 400×600. Why are they 384×384 currently?

**Comment by akshay-iyer** [comment]
Yeah, I had the same doubt regarding the image size of synthetic images not being 400*600. When I'm training on my own dataset, do I have to resize them to 400*400 or can I use them in their original size 512*384
